schema/metrics/history.json

```diff
@@ -139,5 +139,16 @@
                 "redirect-prefix": "/for-the-press"
             }
         ]
+    },
+
+    "digests": {
+        "frames": [
+            {
+                "id": 1,
+                "ends": null,
+                "starts": "2017-09-01",
```

Comment:
this is the 'digests started appearing on the webiste' date and not a published date.

Reply:
Correct me if I'm wrong but the metrics of:

- a digest published on 2017-09-01 and viewed 0 times on 2017-09-01, 42 times on 2017-09-02
- a digest published on 2017-09-02 and viewed 42 times on 2017-09-02

are equivalent, right?

Reply:
(It's a mostly unrelated question)

Reply:
... they are equivalent in terms of total page views, as the dates are inclusive. Did I understand you correctly?

this file is used to describe 'epochs' of url patterns. 2017-09-01 is arbitrary, it's simply a nice round value that happened before digests were/will be introduced. I've gone to some lengths over time to try and capture all the different permutations we've seen. Article metrics are still their own little world but I'd like to migrate them into this file as well